### ISSUE TITLE
Missing config of normalization layer - keras.applications.EfficientNetB0

### ISSUE THREAD

**Issue by ronigober** [keras-team/keras#16392]
**System information**.

- OS Platform and Distribution: Windows 10
- TensorFlow installed from: pip install
- TensorFlow version : 2.8.0
- Python version: 3.8.0
- GPU model and memory: 3060 Ti 8GB

**Describe the problem**.

When using [to_json](https://keras.io/api/models/model_saving_apis/#tojson-method) function for later reloading of tf.keras.application.EfficientNetB0, the layer configuration is missing.

**Describe the current behavior**.

Layer's json configuration is:
{"class_name": "Normalization", "config": {"name": "normalization_1", "trainable": true, "dtype": "float32", "axis": [3], "mean": null, "variance": null}, "name": "normalization_1", "inbound_nodes": [[["rescaling_1", 0, 0, {}]]]}

While in the normalization layer inside model.layers, "mean", "variance", "adapt_mean", "adapt_variance", "non_trainable_weights", "non_trainable_variables", "weights", "non_trainable_variables", are holding the well known mean and variance of imagenet dataset: [0.485, 0.456, 0.406], [0.229, 0.224, 0.225].

In addition, "count" weights of normalization layer are missing too.

**Describe the expected behavior**.

I expect that the right parameters will be exported in config() - self.mean & self.variance.

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**.

- Do you want to contribute a PR? (yes/no): Yes, but not sure, where I need to make this change.

**Standalone code to reproduce the issue**.

from tensorflow.keras.applications import EfficientNetB0
model = tf.keras.applications.efficientnet.EfficientNetB0(weights='imagenet')
json = model.to_json()

**Comment by tilakrayal** [comment]
Hello @ronigober,
In order to expedite the trouble-shooting process, I request you, please provide a complete code you are using.Thanks!

**Comment by ronigober** [comment]
Hi @tilakrayal, Here is the complete code.
Thanks!

```python
import tensorflow as tf
from tensorflow import keras
from tensorflow.keras.applications import EfficientNetB0
from tensorflow.keras.models import Model
from tensorflow.keras.layers import GlobalAveragePooling2D, Dense


def load_cifar_10():
    (x_train, y_train), (x_test, y_test) = keras.datasets.cifar10.load_data()
    y_train = keras.utils.to_categorical(y_train, 10)
    y_test = keras.utils.to_categorical(y_test, 10)
    return x_train, y_train, x_test, y_test


def get_model():
    base_model = EfficientNetB0(
        include_top=False,
        weights='imagenet',
        input_tensor=None,
        input_shape=(32, 32, 3))
    x = base_model.output
    x = GlobalAveragePooling2D()(x)
    predictions = Dense(10, activation='softmax')(x)
    model = Model(inputs=base_model.input, outputs=predictions)
    return model


x_train, y_train, x_test, y_test = load_cifar_10()

efn0 = get_model()
efn0.compile(optimizer='adam', loss="categorical_crossentropy", metrics=["categorical_accuracy"])

json = efn0.to_json()
print(json)

```

**Comment by tilakrayal** [comment]
@ronigober ,
I haven't found any issue while executing the given code.Please find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/692c16ce0758fab71319fbb452f8edff/untitled299.ipynb).

**Comment by benboren** [comment]
@tilakrayal It can be seen that the Normalization layer doesn't hold the right configuration, it holds nulls:
```json
{"name": "normalization", "trainable": true, "dtype": "float32", "axis": [3], "mean": null, "variance": null }
```

**Comment by ronigober** [comment]
@tilakrayal It can be seen that the Normalization layer doesn't hold the right configuration, it holds nulls:
```json
{"name": "normalization", "trainable": true, "dtype": "float32", "axis": [3], "mean": null, "variance": null }
```

**Comment by tilakrayal** [comment]
@gadagashwini ,
I was able to reproduce the issue in tf v2.7, [v2.8](https://colab.research.google.com/gist/tilakrayal/692c16ce0758fab71319fbb452f8edff/untitled299.ipynb) and [nightly](https://colab.research.google.com/gist/tilakrayal/80f904c9648d9830fa5b33c6584614f8/16392.ipynb).Please find the attached gist.

**Comment by tilakrayal** [comment]
@ronigober,
I tried to execute the mentioned code with both EfficientNetB0 and MobilenetV2 and it was executed with the similar output which might be intended. Kindly find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/4d6aec5c036f91561a787d30f2fd59f5/untitled1284.ipynb). Thank you!
